src/mgmt/deploy-powerflexgw-vm.sh

```diff
@@ -0,0 +1,346 @@
+    }
+
+  [[ ${iface} == nameserver ]] && {
+    NameServers=${ip} 
+    echo "echo NameServers=${ip} >> /tmp/ks_post_include.txt"
+    }
+
+  [[ ${iface} == gateway ]] && {
+    Gateway=${ip} 
+    echo "echo Gateway=${ip} >> /tmp/ks_post_include.txt"
+    }
+
+
+  [[ ${iface} == user ]] && echo "echo User=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == password ]] && echo "echo Password=${ip} >> /tmp/ks_post_include.txt"
+
+  [[ ${iface} == smuser ]] && echo "echo SMUser=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == smpassword ]] && echo "echo SMPassword=\'${ip}\' >> /tmp/ks_post_include.txt"
+  [[ ${iface} == smpool ]] && echo "echo SMPool=${ip} >> /tmp/ks_post_include.txt"
+
+  [[ ${iface} == satellite_ip ]] && echo "echo SA_ip=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == satellite_hostname ]] && echo "echo SA_host=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == satellite_org ]] && echo "echo SA_org=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == satellite_activation_key ]] && echo "echo SA_key=${ip} >> /tmp/ks_post_include.txt"
+
+  [[ ${iface} == smproxy ]] && echo "echo SMProxy=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == smproxyuser ]] && echo "echo SMProxyUser=${ip} >> /tmp/ks_post_include.txt"
+  [[ ${iface} == smproxypassword ]] && echo "echo SMProxyPassword=${ip} >> /tmp/ks_post_include.txt"
+
+  [[ ${iface} == enp1s0 ]] && {
+    echo "echo network --activate --onboot=true --noipv6 --device=${iface} --bootproto=static --ip=${ip} --netmask=${mask} --hostname=${HostName} --gateway=${Gateway} --nameserver=${NameServers} --mtu=${mtu} >> /tmp/ks_include.txt"
+    }
+
+  [[ ${iface} == enp2s0 ]] && {
+    echo "echo network --activate --onboot=true --noipv6 --device=${iface} --bootproto=static --ip=${ip} --netmask=${mask} --gateway=${Gateway} --nodefroute --mtu=${mtu} >> /tmp/ks_include.txt"
+    }
+    
+done <<< "$( grep -Ev "^#|^;|^\s*$" ${cfg_file} )"
+} >> /tmp/powerflexgw.ks
+
+cat <<'EOFKS' >> /tmp/powerflexgw.ks
+%end
+
+%post --nochroot --logfile /root/powerflexgw-post.log
+# Copy the files created during the %pre section to /root of the installed system for later use.
+  cp -v /tmp/powerflexgw-pre.log /mnt/sysimage/root
+  cp -v /tmp/ks_include.txt /mnt/sysimage/root
+  cp -v /tmp/ks_post_include.txt /mnt/sysimage/root
+  mkdir -p /mnt/sysimage/root/powerflexgw-ks-logs
+  cp -v /tmp/powerflexgw-pre.log /mnt/sysimage/root/powerflexgw-ks-logs
+  cp -v /tmp/ks_include.txt /mnt/sysimage/root/powerflexgw-ks-logs
+  cp -v /tmp/ks_post_include.txt /mnt/sysimage/root/powerflexgw-ks-logs
+%end
+
+
+%post
+
+exec < /dev/tty8 > /dev/tty8
+chvt 8
+
+(
+  # Source the variables from the %pre section
+  . /root/ks_post_include.txt
+
+  #Configure Chrony
+  sed -i -e "/^server /d" /etc/chrony.conf
+  for ntps in ${NTPServers//,/ }
+  do
+    echo "server ${ntps}" >> /etc/chrony.conf
+  done
+
+  # Create a new user
+  useradd ${User}
+  passwd -f ${User} << EOFPW
+${Password}
+${Password}
```

Comment:
Do we create a user for the gateway VM ? did not see references to it being used anywhere

Reply:
No, will remove this part.